Assets/_BForBoss/_Utility/Scripts/FreeRoamCamera.cs

```diff
@@ -159,19 +159,6 @@ private void Update()
                 _onExitCamera?.Invoke();
             }
 
-            // Hide and lock cursor when right mouse button pressed
```

Comment:
Don't think there's a need for this. 
And setting the lock state in different areas causes more bugs in the long run. So if it's not required here, rather not set it.

Reply:
So because the LockState is set in different places, there could be some bug and inconsitencies when the game state changes. 
e.g. 
While holding down right mouse button -> cursor locks, while free roam camera is on, unpause game,  cursor unlocks/locks. 

Different places changing the lock causes inconsistences

Reply:
This still works, it just doesn't lock the mouse

Reply:
ACtually just monkey test this, should be okay to leave this in.